### PR TITLE
Add shopeepay in PH

### PR DESCRIPTION
**Pull request by yanoandri**
## Description

- https://xendit.atlassian.net/browse/TPI-6742
- Add new payment channel for PH ShopeePay

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Cosmetic change (text changing or color changing)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code (if necessary), particularly in hard-to-understand areas
- [ ] My changes generate no new vulnerabilities
- [ ] I have added tests that prove my fix is effective or that my feature works

## Screenshot
Screen Shot 2022-04-06 at 00 08 03
Screen Shot 2022-04-06 at 00 08 46

### CODE REVIEW

**Review by andykim — COMMENTED** [review]
LGTM.

I see you created a new payment method because of the default config `min/max total & description`

**Review by andykim — APPROVED** [review]

File: Xendit/M2Invoice/etc/config.xml [hunk 1]
```diff
@@ -441,6 +441,20 @@
                 <sort_order>30</sort_order>
                 <currency>PHP</currency>
             </paymaya>
+            <shopeepayph>
+                <active>1</active>
+                <payment_action>initialize</payment_action>
+                <model>Xendit\M2Invoice\Model\Payment\SHOPEEPAYPH</model>
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/etc/config.xml:447]: i think you can reuse the same shopeepay (idr) model

File: Xendit/M2Invoice/view/frontend/layout/multishipping_checkout_billing.xml [hunk 2]
```diff
@@ -38,6 +38,7 @@
                     <item name="bjbva" xsi:type="string">Xendit_M2Invoice::multishipping/bjbva.phtml</item>
                     <item name="bsiva" xsi:type="string">Xendit_M2Invoice::multishipping/bsiva.phtml</item>
                     <item name="cashalo" xsi:type="string">Xendit_M2Invoice::multishipping/cashalo.phtml</item>
+                    <item name="shopeepayph" xsi:type="string">Xendit_M2Invoice::multishipping/shopeepayph.phtml</item>
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/view/frontend/layout/multishipping_checkout_billing.xml:41]: you can also reuse the shopeepay.phtml as well

**Review by IreneGohtami — APPROVED** [review]
some comments, but else lgtm

**Pull request merged**